scripts/apps/authoring-react/authoring-integration-wrapper.tsx

```diff
@@ -241,6 +241,7 @@ interface IPropsWrapper extends IProps {
 interface IState {
     sidebarMode: boolean | 'hidden';
     sideWidget: null | ISideWidget;
+    sideWidgetSecondary: null | ISideWidget;
```

Comment:
According the the interface it's possible for both widgets to be pinned at once. I'm not requesting to fix it - rather keep in mind in the future to avoid similar situations when possible.

Reply:
![image](https://github.com/user-attachments/assets/c8fcd31e-5742-484f-a05b-000ca476b467)

this would be better for example